src/algebra/algebra/subalgebra.lean

```diff
@@ -960,6 +960,29 @@ lemma smul_mem_pointwise_smul (m : R') (r : A) (S : subalgebra R A) : r ∈ S 
 
 end pointwise
 
+section center
+
+variables (R A)
+
+/-- The center of an algebra is the set of elements which commute with every element. They form a
+subalgebra. -/
+def center : subalgebra R A :=
+{ algebra_map_mem' := by simp [algebra.commutes, subsemiring.mem_center_iff],
```

Comment:
I'd be tempted to extract a `set.algebra_map_mem_center` here, since we did that for 0 / 1 / + / etc

Reply:
OK done.